src/FakeItEasy.netstd/FakeItEasy.netstd.nuget.targets

```diff
@@ -1,9 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8" standalone="no"?>
-<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup Condition="'$(NuGetPackageRoot)' == ''">
-    <NuGetPackageRoot>$(UserProfile)\.nuget\packages\</NuGetPackageRoot>
-  </PropertyGroup>
-  <ImportGroup>
-    <Import Project="$(NuGetPackageRoot)\StyleCop.MSBuild\4.7.54\build\StyleCop.MSBuild.Targets" Condition="Exists('$(NuGetPackageRoot)\StyleCop.MSBuild\4.7.54\build\StyleCop.MSBuild.Targets')" />
-  </ImportGroup>
-</Project>
```

Comment:
There's also an entry in the csproj for this file which will have to be removed:

https://github.com/FakeItEasy/FakeItEasy/blob/3756a07a38b2c02347b72359fdecae70ea2377a7/src/FakeItEasy.dotnet/FakeItEasy.dotnet.csproj#L617-L619